db/client.ts

```diff
@@ -1,5 +1,5 @@
-import { drizzle } from "drizzle-orm/vercel-postgres";
-import { sql } from "@vercel/postgres";
+import { neon } from "@neondatabase/serverless";
+import { drizzle } from "drizzle-orm/neon-http";
 
-// Use this object to send drizzle queries to your DB
+const sql = neon(process.env.DATABASE_URL!);
```

Comment:
whats the difference?

Reply:
supabase might still be a better option but if this branching works that'll be cool

Reply:
nice, supabase has branching too

Reply:
but neon will probably have better vercel integration